content/cli/cloud/secret.md

```diff
@@ -52,25 +52,24 @@ This will delete the named secret.
 fluvio cloud secret delete <NAME>
 ```
 
-
 ## Connector config file references
 
-The connector config files can reference cloud secrets by NAME generically as follows:
+The connector config files can reference cloud secrets by NAME. They need to be referenced on meta section of connector config. And then we can use the secret name in the connector configuration parameters. The secret can be used in the configuration as `${{ secrets.<NAME> }}`.
 
 ```yaml
+apiVersion: 0.1.0
 meta:
-  version: 0.1.0
+  version: 0.2.0
   name: my-connector
   type: package-name
   topic: a-topic
-  create-topic: true
-<CUSTOM>:  # named section for custom config parameters, usually a short name like "http", or "mqtt"
-  param_client_id: 
-    secret:
-      name: CAT_FACTS_CLIENT_ID
-  param_client_secret:
-    secret: 
-      name: CAT_FACTS_SECRET
+  secrets:
+    - name: CAT_FACTS_CLIENT_ID
+    - name: CAT_FACTS_SECRET
+# named section for custom config parameters, usually a short name like "http", or "mqtt"
+<CUSTOM>:  
+  param_client_id: ${{ secrets.CAT_FACTS_CLIENT_ID }}
+  param_client_secret: ${{ secrets.CAT_FACTS_SECRET }}
```

Comment:
Do we only accept interpreted syntax for secrets, or is whole "secrets:\nname: SECRET_NAME" still accepted?

Reply:
that is still accepted. I could add both options to the example. Not sure if we should incentivate our users to use the string interpolation option

Reply:
I think we should document what we accept. If we decide to remove it, then we can note the deprecation.

Reply:
One configuration please! Let's not add options for the same thing. I will create unnecessary confusion. It will also increase the maintenance work for us.

Reply:
We should file an issue to remove that form of it then, not urgent though...

Reply:
As this is a new feature, that we are documenting now, we should only document what we want to support. I'm less concerned about what the code does (as an alternative approach).